rpcs3/rpcs3qt/emu_settings.cpp

```diff
@@ -225,30 +225,39 @@ void emu_settings::EnhanceComboBox(QComboBox* combobox, emu_settings_type type,
 	const QString selected_q = qstr(selected);
 	int index = -1;
 
-	if (is_ranged)
-	{
-		index = combobox->findData(selected_q);
-	}
-	else
+	// Since the QComboBox has localised strings, we can't just findText / findData, so we need to manually iterate through it to find our index
+	auto find_index = [&](const QString& value)
 	{
+		int idx = -1;
 		for (int i = 0; i < combobox->count(); i++)
 		{
 			const QVariantList var_list = combobox->itemData(i).toList();
 			ASSERT(var_list.size() == 2 && var_list[0].canConvert<QString>());
 
 			if (selected_q == var_list[0].toString())
 			{
-				index = i;
+				idx = i;
 				break;
 			}
 		}
+		return idx;
+	};
+
+	if (is_ranged)
+	{
+		index = combobox->findData(selected_q);
+	}
+	else
+	{
+		index = find_index(selected_q);
 	}
 
 	if (index == -1)
 	{
 		const std::string def = GetSettingDefault(type);
 		cfg_log.error("EnhanceComboBox '%s' tried to set an invalid value: %s. Setting to default: %s", cfg_adapter::get_setting_name(type), selected, def);
-		combobox->setCurrentIndex(combobox->findData(qstr(def)));
+		index = find_index(qstr(def));
```

Comment:
This will not work for ranged lists...
Use this:
```suggestion
		if (is_ranged)
		{
			index = combobox->findData(qstr(def));
		}
		else
		{
			index = find_index(qstr(def));
		}
```